### PR TITLE
Sanitises flavour text

### LINKED ISSUE #9530: ´ character breaks examining and a bunch of other stuff
**Problem Description**:
People with a ´ in their flavourtext will break a whole bunch of admin verbs, examining etc.

**What did you expect to happen**: 
To be able to examine them, use VV, player panel etc.

**What happened instead**:
Examine won't work, other admin verbs won't work @Spartan6 experienced this

**Why is this bad/What are the consequences:**
It's a bug
We had to use Adv Proc-call to VV them...

**Steps to reproduce the problem**:
Add a ´ into flavour text

### PR DESCRIPTION
**Pull request by Birdtalon**
Sanitises flavor text.

fixes #9530 

🆑 Birdtalon
fix: Flavor text is now sanitised to avoid examine problems.
\🆑

### CODE REVIEW

File: code/__HELPERS/text.dm [hunk 1]
```diff
@@ -63,6 +63,10 @@
 	var/regex/alphanum_only = regex("\[^a-zA-Z0-9# ,.?!:;()]", "g")
 	return alphanum_only.Replace(t, "#")
 
+/proc/multiline_paranoid_sanitize(txt)
+	var/regex/alphanum_only = regex("\[^a-zA-Z0-9# ,.?!:;\\r\\n()]", "g")
```
Comment (Birdtalon) [inline code/__HELPERS/text.dm]: This actually doesn't work and will still filter newlines. Could anyone offer a solution?

**Comment by tigercat2000** [comment]
Additionally, replacing the invalid characters with spaces would be more appropriate than `#` - the reason the other sanitization proc does that is because it's spit into a terminal directly and spaces could cause issues.

**Comment by Birdtalon** [comment]
@tigercat2000 will do.

**Comment by FlattestGuitar** [comment]
Hey its me ur fursona.

Plenty of people put links to imgur in their flavor text. This will break that. An obvious character here is `/`, but there could be more characters that somehow end up in a URL that I can't think of right now.

**Comment by tigercat2000** [comment]
@Birdtalon Ah, didn't notice that. However, still, an ascii character definitely should work :stuck_out_tongue:

**Comment by Ty-Omaha** [comment]
will this make apostrophes show up as `&#39;`?

**Comment by Birdtalon** [comment]
Re-opening with a Regex suggestion from Tigercat. I can't test it until later today however.

**Pull request closed without merging**